src/librustc_driver/driver.rs

```diff
@@ -161,23 +174,20 @@ pub fn compile_input(sess: &Session,
                            "indexing hir",
                            move || hir_map::map_crate(hir_forest, defs));
 
-
-        write_out_deps(sess, &outputs, &id);
-
         {
             let _ignore = hir_map.dep_graph.in_ignore();
-            controller_entry_point!(after_write_deps,
+            controller_entry_point!(after_ast,
```

Comment:
I think `after_hir_lowering` might be a better name here - `ast` is a data structure rather than a phase.